guides/source/association_basics.md

```diff
@@ -2661,6 +2661,18 @@ Extensions can refer to the internals of the association proxy using these three
 * `proxy_association.reflection` returns the reflection object that describes the association.
 * `proxy_association.target` returns the associated object for `belongs_to` or `has_one`, or the collection of associated objects for `has_many` or `has_and_belongs_to_many`.
 
+### Association Scoping using the Association Owner
+
+The owner of the association can be passed the owner as a single argument to the
+scope block in situations where you need even more control over the association
```

Comment:
Maybe mention that this makes preloading impossible, as it is an important caveat.